portal/models/user.py

```diff
@@ -1643,8 +1647,19 @@ def check_role(self, permission, other_id):
         returns true if permission should be granted, raises 404 if the
         other_id can't be found, otherwise raise a 401
 
+        NB - a user with "url_authenticated" as their current encounter's
+        auth_type will NOT have any access, unless specifically requested
+        via the "allow_on_url_authenticated_encounters" parameter
+
         """
         assert (permission in ('view', 'edit'))  # limit vocab for now
+        if (
+                not allow_on_url_authenticated_encounters and
+                current_app.config.get('ENABLE_URL_AUTHENTICATED') and
```

Comment:
Is there a `os.environ.get('ENABLE_URL_AUTHENTICATED')` call in [config.py](https://github.com/uwcirg/truenth-portal/blob/develop/portal/config/config.py) for this? (if we want to config via docker)

Reply:
i expect this to become part of eproms' site_persistence config, but given they might want to toggle before a release cycle - yes, we'll need that!

Reply:
Added in 90156217c91cb1b151bc6834f99ff6f2b22256fc